tests/cmd/cmd_test.go

```diff
@@ -0,0 +1,96 @@
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/kylelemons/godebug/diff"
+)
+
+var Fixtures = os.ExpandEnv("$GOPATH/src/github.com/kedgeproject/kedge/tests/cmd/fixtures/")
+var ProjectPath = "$GOPATH/src/github.com/kedgeproject/kedge/"
+var BinaryLocation = os.ExpandEnv(ProjectPath + "kedge")
+
+func TestKedgeGenerate(t *testing.T) {
+	testCases := []struct {
+		name        string
+		command     []string
+		input       string
+		wantSuccess bool
+		error       string
+	}{
+		{
+			name:        "multiple configmaps given and name not specified",
+			command:     []string{"generate", "-f=" + Fixtures + "multi-configmapname/app.yaml"},
```

Comment:
this test just generates an all the files are always in `Fixtures` dir.
It would be more readable to specify just file/dir path.
```
	testCases := []struct {
 		name        string
 		command     path
 		input       string
 		wantSuccess bool
 		error       string
 	}
```

```
name: "multiple configmaps given and name not specified",
path:  "multi-configmapname/app.yaml"
..
```

proper command can be build inside for loop before test is run

Reply:
done!